examples/tpm_test.h

```diff
@@ -43,6 +42,8 @@
 
 #define TPM2_DEMO_NV_TEST_INDEX         0x01800200
 #define TPM2_DEMO_NV_TEST_AUTH_INDEX    0x01800201
+#define TPM2_DEMO_NVRAM_STORE_PUB_INDEX 0x01800202
```

Comment:
The pub index is not used... Please remove and rename `TPM2_DEMO_NVRAM_STORE_PRIV_INDEX`, since it contains both private and public portions.

Reply:
fixed. Thanks